app/assets/stylesheets/pageflow/progress_navigation_bar.css.scss

```diff
@@ -15,53 +15,13 @@ $nav-active-chapter: #6D772E;
 
 $progressbar-mobile-width: 14px;
 
+$header-button-enabled: true;
```

Comment:
Has to be moved to themes/default.scss and unified with $nav-display-header-button`